mmv1/templates/terraform/examples/app_engine_flexible_app_version.tf.erb

```diff
@@ -17,6 +17,11 @@ resource "google_project_service" "service" {
   disable_dependent_services = false
 }
 
+resource "google_service_account" "custom_service_account" {
+  account_id   = "<%= ctx[:vars]['account_id'] %>"
```

Comment:
Only the `TestAccAppEngineFlexibleAppVersion_appEngineFlexibleAppVersionExample` is related to this PR.

Error is:

```
Error: "account_id" ("") doesn't match regexp "^[a-z](?:[-a-z0-9]{4,28}[a-z0-9])$"
```

It looks like this variable isn't set. You'll need to set a value in terraform.yaml - https://github.com/GoogleCloudPlatform/magic-modules/blob/73e52c105fc402d5f6ba58c8cd42b1eab17185b6/mmv1/products/appengine/terraform.yaml#L146

I would recommend `my-account` as the value to be consistent with similar tests, but the only restriction is that it needs to contain a hyphen.

Reply:
updated to 'my-account', still seeing test failure.
Is there any way for me to get access to these test logs?

Reply:
Current error message:

```
entral1/operations/90f99aac-b5cb-4b02-a321-7742329ee007 error [INTERNAL]: An internal error occurred while processing task /app-engine-flex/insert_flex_deployment/flex_create_resources>2022-08-10T09:40:10.663Z237364.jc.2: Deployment Manager operation tf-test-appeng-flexegjuscsmh3/operation-1660124411893-5e5dfd5da9556-8cff1117-a83474d8 errors: [code: "RESOURCE_ERROR"
  | location: "/deployments/aef-default-v1/resources/aef-default-v1-00"
  | message: "{\"ResourceType\":\"compute.beta.regionInstanceGroupManager\",\"ResourceErrorCode\":\"SERVICE_ACCOUNT_ACCESS_DENIED\",\"ResourceErrorMessage\":\"The user does not have access to service account \'One of [projects/{project-id}/serviceAccounts/my-account@{project-id}.iam.gserviceaccount.com]\'.  User: \'{project-number}@cloudservices.gserviceaccount.com\'.  Ask a project owner to grant you the iam.serviceAccountUser role on the service account\"}"
  | ]
```

No, there is no way for you to get access to those logs. But you can [run tests locally](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) to reproduce the error and then make sure it's fixed prior to pushing any changes.

You can get the service account that is running the tests by [setting a test_env_var in terraform.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/06fae4768a1708f40dc4b3468e2da1964d0896ec/mmv1/products/notebooks/terraform.yaml#L59) and [referencing it in the example](https://github.com/GoogleCloudPlatform/magic-modules/blob/06fae4768a1708f40dc4b3468e2da1964d0896ec/mmv1/templates/terraform/examples/notebook_instance_full.tf.erb#L12). (You can set a service account locally with the GOOGLE_SERVICE_ACCOUNT env var.)